build/h3text.2018

```diff
@@ -24,7 +24,7 @@ NET :  10.0.0.0 : ARPANET :
 NET : 172.0.0.0 : B-172 :
 NET : 192.0.0.0 : C-192 :
 
-HOST : CHAOS 177001 : BRIDGE : UNIX : UNIX : :
+HOST : CHAOS 177001 : BRIDGE, LOCAL-TIME-SERVER : UNIX : UNIX : :
```

Comment:
I think it would be useful with a comment in the file, saying essentially "please make sure to update this address if/when you join the global Chaosnet"?

Reply:
Good idea. I’ll update.